components/brave_today/browser/suggestions_controller.cc

```diff
@@ -41,6 +41,8 @@ constexpr double kSimilarSubscribedMax = 0.4;
 constexpr double kSimilarVisitedMin = 0;
 constexpr double kSimilarVisitedMax = 0.3;
 
+constexpr uint64_t kMaxSuggestions = 15;
+
```

Comment:
nit: If this variable is used only by the new code, how about moving this variable into the scope?